uvm/api/com/untangle/uvm/LocalDirectoryUser.java

```diff
@@ -129,7 +148,7 @@ public String getPasswordBase64Hash() {
      * @param passwordBase64Hash The hash
      */
     public void setPasswordBase64Hash(String passwordBase64Hash) {
-        this.passwordBase64Hash = makeNotNull(passwordBase64Hash);
+        this.passwordBase64Hash = passwordBase64Hash;
```

Comment:
We don't need `makeNotNull()` ?

Reply:
Yes, cause when we encrpt the actual password we need to set this field null.